pom.xml

```diff
@@ -267,12 +267,12 @@
 		<uiframeworkVersion>3.8</uiframeworkVersion>
 		<kenyacoreVersion>2.4.1</kenyacoreVersion>
 		<kenyauiVersion>2.1.1</kenyauiVersion>
-		<kenyaemrVersion>17.0.3</kenyaemrVersion>
+		<kenyaemrVersion>18.0.0</kenyaemrVersion>
 		<jodaTimeVersion>2.9.1</jodaTimeVersion>
 		<webservices.restVersion>2.1</webservices.restVersion>
 		<uilibraryVersion>2.0.4</uilibraryVersion>
 		<hospitalcoreVersion>1.0.7-SNAPSHOT</hospitalcoreVersion>
-		<ehrconfigsVersion>1.0.0</ehrconfigsVersion>
-		<legacyUiVersion>1.5.0</legacyUiVersion>
+		<ehrconfigsVersion>1.0.10-SNAPSHOT</ehrconfigsVersion>
+		<legacyUiVersion>1.8.2</legacyUiVersion>
```

Comment:
Is 1.5.0 of the legacy UI not working for you to change to this version, are you getting any errors with that lower version as you test?

Reply:
I thought it would be prudent to base to versions currently running here https://github.com/palladiumkenya/kenyahmis-releases/releases/tag/v47

Reply:
Let depend on what they are depending on an NOT what they are running see this https://github.com/palladiumkenya/openmrs-module-kenyaemr/blob/v18.0.0/pom.xml#L74